components/counter-bar/src/index.js

```diff
@@ -63,166 +61,172 @@ const Counter = styled('button')(
- *    { id: 'jkl', name: 'Counter 4', score: 9 },
- *    { id: 'mno', name: 'Counter 5', score: 2 },
- *    { id: 'pqr', name: 'Counter 6', score: 1 },
- *    { id: 'stu', name: 'Counter 7' },
- *  ]}
- *  // eslint-disable-next-line no-alert
- *  onSelect={ (id) => alert(`Counter with id: ${id} selected.`) }
- * />
+ * <CounterBar>
+ *   <CounterBar.Total score={16}>All counters</CounterBar.Total>
+ *   <CounterBar.Container>
+ *     <CounterBar.Item score={0}>Counter 1</CounterBar.Item>
+ *     <CounterBar.Item score={2} active onSelect={ (e) => alert(`${e.target} clicked`) }>Counter 2</CounterBar.Item>
+ *     <CounterBar.Item score={2}>Counter 3</CounterBar.Item>
+ *     <CounterBar.Item score={9}>Counter 4</CounterBar.Item>
+ *     <CounterBar.Item score={2}>Counter 5</CounterBar.Item>
+ *     <CounterBar.Item score={1}>Counter 6</CounterBar.Item>
+ *     <CounterBar.Item>Counter 7</CounterBar.Item>
+ *   </CounterBar.Container>
+ * </CounterBar>
  * ```
- * 
- * Padded container
- * ```jsx 
- * <CounterBar
- *  listTitle="All counters"
- *  name="name"
- *  counters={[
- *    { id: 'abc', name: 'Counter 1', score: 0 },
- *    { id: 'def', name: 'Counter 2', score: 2 },
- *    { id: 'ghi', name: 'Counter 3', score: 2 },
- *    { id: 'jkl', name: 'Counter 4', score: 0 },
- *    { id: 'mno', name: 'Counter 5', score: 2 },
- *    { id: 'pqr', name: 'Counter 6', score: 0 },
- *    { id: 'stu', name: 'Counter 7', score: 2 },
- *  ]}
- * />
+ * CounterBar with padded container
+ * ```jsx
+ * <CounterBar>
+ *   <CounterBar.Total score={8}>All counters</CounterBar.Total>
+ *   <CounterBar.Container>
+ *     <CounterBar.Item score={0}>Counter 1</CounterBar.Item>
+ *     <CounterBar.Item score={2}>Counter 2</CounterBar.Item>
+ *     <CounterBar.Item score={2}>Counter 3</CounterBar.Item>
+ *     <CounterBar.Item score={0}>Counter 4</CounterBar.Item>
+ *     <CounterBar.Item score={2}>Counter 5</CounterBar.Item>
+ *     <CounterBar.Item score={0}>Counter 6</CounterBar.Item>
+ *     <CounterBar.Item score={2}>Counter 7</CounterBar.Item>
+ *   </CounterBar.Container>
+ * </CounterBar>
  * ```
- * 
  * Zero/no scores
- * ```jsx 
- * <CounterBar
- *  listTitle="All counters"
- *  name="name"
- *  counters={[
- *    { id: 'abc', name: 'Counter 1', score: 0 },
- *    { id: 'def', name: 'Counter 2' },
- *    { id: 'ghi', name: 'Counter 3', score: 0 },
- *  ]}
- * />
+ * ```jsx
+ * <CounterBar>
+ *   <CounterBar.Total score={0}>All counters</CounterBar.Total>
+ *   <CounterBar.Container>
+ *     <CounterBar.Item score={0}>Counter 1</CounterBar.Item>
+ *     <CounterBar.Item>Counter 2</CounterBar.Item>
+ *     <CounterBar.Item score={0}>Counter 3</CounterBar.Item>
+ *   </CounterBar.Container>
+ * </CounterBar>,
  * ```
  */
-const CounterBar = ({ activeTitle, listTitle, counters, onSelect, ...props }) => {
-  const totalScore = counters.reduce((a, b) => a + (Number(b.score) || 0), 0);
+const CounterBar = ({ children, ...props }) => {
```

Comment:
this isn't really needed

as it is, this could be rewritten as just:
```
const CounterBar = props => <Wrapper {...props />;
```
and maintain identical functionality

Reply:
Good call - very nice - I will apply this elsewhere, too.

Reply:
or indeed `const CounterBar = Wrapper;`

Reply:
This is great and I would use it, but actually the automatic documentation doesn't recognise it, so I'm going with `const CounterBar = props => <Wrapper {...props}/>`.